source/crud/tests/v2/bulkWrite-update-validation.yml

```diff
@@ -0,0 +1,75 @@
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+  - {_id: 3, x: 33}
+
+tests:
+  -
+    description: "BulkWrite replaceOne prohibits atomic modifiers"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              name: "replaceOne"
+              arguments:
+                filter: { _id: 1 }
+                # Only the first field is tested, as the spec permits drivers to
+                # only check that and rely on the server to check subsequent
+                # fields.
+                replacement: { $set: { x: 22 }}
+        error: true
+    expectations: []
```

Comment:
Some drivers may only consider `expectations` a subset of observed events, in which case this will have no effect, but for others this can be used to assert that no commands were executed and that the error was reported entirely on the client side. Existing "clientError" tests do this.